### DISCUSSION TITLE
Integrating simbrief plans without Navigraph?

### DISCUSSION THREAD

**Discussion by c-goldberg** [flybywiresim/aircraft#7041] [Help] [answered]
I’m having trouble loading a simbrief plan into the fbw a320 MCDU. I can create my plan in simbrief and then load it into the flypad just fine. I also input my simbrief ID into the MCDU. When I follow the instructions to [request data from [simbrief],](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/simbrief/#request-data-from-simbrief) I can also see my departure and arrival airports, but when I try to initialize the request in INIT A, it says "not in database."
![Untitled1](https://user-images.githubusercontent.com/39924670/162650707-217b1f45-8959-45e9-b087-702719b5ea49.jpg)

[The FBW tutorial for flight planning](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preflight/) mentions that out of date AIRACs may result in “not in database” errors. The default simbrief AIRAC is 2003, which is in fact out of date. Searches online seem to turn up results indicating that only Navigraph subscribers can access up to date AIRAC info on simbrief.
![Untitled](https://user-images.githubusercontent.com/39924670/162650715-3428baac-e611-4f06-886f-ef006cb7f810.jpg)

Basic question: Can I import a simbrief plan into the fbw a320 without a Navigraph subscription? Am I missing something really simple here?

**Comment by tracernz** [comment]
If you can use the #a32nx-support channel on Discord that will work better. May be an issue with sim navdata rather than out of date AIRAC on simbrief (although that can always be an issue).

**Comment by eljiot5169** [comment, marked as answer]
In msfs LFTH doesnt exist so its not in Database. You will need an addon.

**Reply by tracernz** [reply]
Ah, good point... indeed with default navdata + scenery it doesn't exist.

With Navigraph navdata it's there:
![image](https://user-images.githubusercontent.com/9995998/162666090-0f9c1948-bbbf-4ef6-8732-10ef6fa28a6c.png)

**Reply by c-goldberg** [reply]
Can confirm. Chose different destination and all good. Thanks for help!
